client/post-editor/post-editor.jsx

```diff
@@ -9,6 +9,7 @@ import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import { localize } from 'i18n-calypso';
 import classNames from 'classnames';
+import tinyMCE from 'tinymce/tinymce';
```

Comment:
Convention is to camelcase even acronyms / abbreviations: `tinyMce`

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions